dask_cuda/cli.py

```diff
@@ -5,21 +5,28 @@
 import click
 from tornado.ioloop import IOLoop, TimeoutError
 
-from dask import config
+from dask import config as dask_config
+from distributed import Client
 from distributed.cli.utils import install_signal_handlers
 from distributed.preloading import validate_preload_argv
 from distributed.security import Security
 from distributed.utils import import_term
 
-from ..cuda_worker import CUDAWorker
+from .cuda_worker import CUDAWorker
+from .utils import print_cluster_config
 
 logger = logging.getLogger(__name__)
 
 
 pem_file_option_type = click.Path(exists=True, resolve_path=True)
 
 
-@click.command(context_settings=dict(ignore_unknown_options=True))
+@click.group
+def cuda():
+    """GPU subcommands."""
```

Comment:
Was thinking a more detailed description like

```suggestion
    """Subcommands to launch or query distributed workers with GPUs."""
```

Could be nice here but happy to hear feedback